### PR TITLE
Update CollectionItem.js

### PR DESCRIPTION
**Pull request by EGedeon**
className support for possible

```
class="collection-item avatar"
```

### CODE REVIEW

File: src/CollectionItem.js [hunk 1]
```diff
@@ -7,21 +7,23 @@ class CollectionItem extends React.Component {
     this.isAnchor = this.isAnchor.bind(this);
   }
   render() {
+    let {active, className} = this.props;
```
Comment (alextrastero) [inline src/CollectionItem.js]: tiny, could be const.

Comment (EGedeon) [inline src/CollectionItem.js]: So, imposibble to set className staying? :)

Comment (isaiah) [inline src/CollectionItem.js]: Can you setup the exlint rules to enforce this?

Comment (isaiah) [inline src/CollectionItem.js]: will refactor all once we have this enforced. merging.

**Pull request merged**